Cisco/iOS-XE/snmp.conf

```diff
@@ -0,0 +1,9 @@
+! SNMP IOS-XE
+!
+! ACL for SNMP Access
+access-list 33 remark SNMP Query ACL
```

Comment:
Suggested change (see next comment in review for US and EU clarification)
`access-list 33 remark Kentik SNMP Polling ACL US and EU`